core/src/main/scala/caliban/parsing/parsers/StringParsers.scala

```diff
@@ -104,9 +104,9 @@ private[caliban] trait StringParsers {
       case _                           => l1
     }
     // remove start lines that are only whitespaces
-    val l3           = l2.dropWhile("[ \t]*".r.replaceAllIn(_, "").isEmpty)
+    val l3           = l2.dropWhile("[ \t]*".r.replaceAllIn(_, "").isEmpty): @noinline
```

Comment:
Can you add a comment explaining why noinline?

Reply:
Scala 2.12 (only) inlines the `dropWhile` method and a compiler error is raised because of it. I'll add that in